src/Middleware/RateLimiting/src/NoLimiter.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Threading.RateLimiting;
+
+namespace Microsoft.AspNetCore.RateLimiting;
+internal class NoLimiter<TResource> : PartitionedRateLimiter<TResource>
+{
+    public override int GetAvailablePermits(TResource resourceID)
+    {
+        return 1;
```

Comment:
@BrennanConroy reasonable?

Reply:
I'm using `NoLimiter` as the default for `RateLimitingOptions`

Reply:
Sure, no one will be using this and it's temporary 👍